compact/central_beampipe.xml

```diff
@@ -1,3 +1,9 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Wouter Deconinck -->
+
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Wouter Deconinck -->
```

Comment:
Most top-level `compact/*.xml` files have this doubling.
```suggestion
<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
<!-- Copyright (C) 2022 Wouter Deconinck -->
```

Reply:
Not sure why. I'll fix that later.

Reply:
Fixed... but it required a force push and I lost one commit. On the plus side, the names are more accurate now too.